userdata/rundsc.ps1

```diff
@@ -627,10 +661,12 @@ if ($rebootReasons.length) {
           $priorityClass = $gwProcess.PriorityClass
           $gwProcess.PriorityClass = [Diagnostics.ProcessPriorityClass]::AboveNormal
           Write-Log -message ('process priority for generic worker altered from {0} to {1}.' -f $priorityClass, $gwProcess.PriorityClass) -severity 'INFO'
+          Stop-Service $UpdateService
         }
       }
     }
   }
 }
+Write-Log -message "Disabling Windows Update service"
```

Comment:
this seems out of place here. can we get it inside the code block that does the disable?